source/cloud/aws/sagemaker.md

```diff
@@ -54,6 +53,12 @@ echo "kernel install completed"
 EOF
 ```
 
+```{warning}
+RAPIDS `>24.12` will not be installable on SageMaker Notebook Instances until those instances support
+Amazon Linux 2023 or other Linux distributions with GLIBC of at least 2.28.
+For more details, see https://github.com/rapidsai/deployment/issues/520.
+```
```

Comment:
@jacobtomlinson what do you think about this language for a warning?

I checked the preview and looks like it's rendering OK:

<img width="1497" alt="image" src="https://github.com/user-attachments/assets/262b9a9c-b7e2-4e97-843f-6ae8e53c8055" />

https://rapids-deployment--504.org.readthedocs.build/en/504/cloud/aws/sagemaker/

Reply:
Language looks good 👍